src/main/java/com/project/psk/WebConfig.java

```diff
@@ -0,0 +1,18 @@
+package com.project.psk;
```

Comment:
there already exists a package for configurations called config, maybe you could move it there